sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/automl_image.py

```diff
@@ -109,10 +159,14 @@ def set_limits(
         max_trials: Optional[int] = None,
         timeout_minutes: Optional[int] = None,
     ) -> None:
-        """Limit settings for all AutoML Image Verticals.
-
+        """Limit settings for all AutoML Image Jobs.
+        :keyword max_concurrent_trials: Maximum number of trials to run concurrently.
+        :type max_concurrent_trials: Optional[int]
+        :keyword max_trials: Maximum number of trials to run.
```

Comment:
should we add default values here as well?

Reply:
Added